### ISSUE TITLE
Secret Promo Numbers

### ISSUE THREAD

**Issue by haec0007** [SWTCG/SWTCG-LACKEY#46]
A hidden numbering system for Subordinates seems to have been agreed upon in #19 . Should a hidden numbering system also be implemented for promo cards? We could just continue the numbering system WOTC started and fill in the gaps they missed.

**Comment by Matt-Ferus** [comment]
That would work.
Filling in the WotC promo gaps sounds....interesting actually.
Would you just choose the earliest promos to fill in the gaps?

**Comment by haec0007** [comment]
Yeah, that was my thought. The earliest two promos could get numbers 27 and 28, then the rest would continue on starting with number 37.

**Comment by Matt-Ferus** [comment]
Would we eventually officially add the WotC promos to Lackey, too? (including the not-released Obi-Wan one)?

**Comment by haec0007** [comment]
Great minds think alike. That was on my mind too. I have Obi-Wan done, so we could add him now or wait until more are ready.

**Comment by MatthewLarson** [comment]
I like this.

**Comment by ChrisMoExp** [comment]
> 
> 
> Great minds think alike. That was on my mind too. I have Obi-Wan done, so we could add him now or wait until more are ready.

@haec0007 when you say you have the Obi-Wan Promo from WOTC done, do you mena the Obi-Wan (F) promo? and is this based on any knowledge you have of the card.... If there is _any_ knowledge of that card i an hungry for it, it is the only one I don't have.

**Comment by ChrisMoExp** [comment]
> 
> 
> Yeah, that was my thought. The earliest two promos could get numbers 27 and 28, then the rest would continue on starting with number 37.

i'd say one day we could, in our wildest dreams admitedly, find out there was a draft for 27 and 28 so i'd propose leaving those alone and leave it as an oddity that there is a gap. That oddity is CAF.

**Comment by haec0007** [comment]
> > Great minds think alike. That was on my mind too. I have Obi-Wan done, so we could add him now or wait until more are ready.
> 
> @haec0007 when you say you have the Obi-Wan Promo from WOTC done, do you mena the Obi-Wan (F) promo? and is this based on any knowledge you have of the card.... If there is _any_ knowledge of that card i an hungry for it, it is the only one I don't have.

Yes, I found a very low-res image of the Obi-Wan (F) promo online and recreated it in high resolution. I don't believe the actual card exists though.
